.circleci/config.yml

```diff
@@ -2913,9 +3012,19 @@ workflows:
           name: "PHP 81 Xdebug tests"
           docker_image: "datadog/dd-trace-ci:php-8.1_buster"
           xdebug_version_one: "3.1.0"
+      - xdebug_tests:
+          requires: [ 'Prepare Code' ]
+          name: "PHP 82 Xdebug tests"
+          docker_image: "datadog/dd-trace-ci:php-8.2_buster"
+          xdebug_version_one: "3.1.0"
```

Comment:
I don't know what `xebug_version_one` is for, exactly. In `dockerfiles/ci/buster/build-extensions.sh` you pull 3.2.0-rc.X. Not a blocker for merging, but I'm curious why.

Reply:
Nah, good catch, it's wrong.